### PR TITLE
added dbt-feast-teradata pipeline

### PR DESCRIPTION
**Pull request by chillanki**
New getting started page has been added.

### CODE REVIEW

File: modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc [hunk 1]
```diff
@@ -0,0 +1,310 @@
+= A Data pipeline with DBT+FEAST on Teradata
+
+:experimental:
+:page-author: Ravi Chr
```
Comment (adamtwo) [inline modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc:4]: Is that your real name :)?

File: modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+:page-email: ravi.chillanki@teradata.com
+:description: DBT Feast integration with Teradata
```
Comment (adamtwo) [inline modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc:6]: DBT is spelled with all lower-case letters

File: modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+=== DBT
+link:https://www.getdbt.com/product/what-is-dbt/[dbt] (Data Build Tool) is a data transformation tool that is the cornerstone of the Modern Data Stack. It takes care of the T in ELT (Extract Load Transform). The assumption is that some other process brings raw data into your data warehouse or lake. This data then needs to be transformed.
+
+=== FEAST
+link:https://docs.feast.dev/[Feast] (Feature Store) is a flexible data system that utilizes existing technology to manage and provide machine learning features to real-time models. It allows for customization to meet specific needs. It also allows us to make features consistently available for training and serving, avoid data leakage and decouple ML from data infrastructure.
+
+
+== Prerequisites
+
+* Access to a Teradata Vantage database instance.
```
Comment (adamtwo) [inline modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc:23]: Many users do not have access to Vantage for experimentation purposes. We like to point them to ClearScape Analytics Experience with the following include:

```
include::ROOT:partial$vantage_clearscape_analytics.adoc[]
```

File: modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+
+* Feast-Teradata library installed locally. See link:https://quickstarts.teradata.com/modelops/using-feast-feature-store-with-teradata-vantage.html#_overview[Feast-Teradata Installation instructions]
+
+* DBT library installed locally. See link:https://docs.getdbt.com/docs/get-started/installation[DBT Installation instructions]
```
Comment (adamtwo) [inline modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc:27]: We should point users to our own tutorial instead as it's tailored for Teradata: modules/ROOT/pages/dbt.adoc

File: modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+----
+python3 -m venv env
+source env/bin/activate
+----
+
+2. Clone the tutorial repository and cd into the project directory:
++
+[source, bash]
+----
+git clone https://github.td.teradata.com/rc255085/Tdatapipeline.git
```
Comment (adamtwo) [inline modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc:46]: I'm assuming this repo needs to be moved to github.com before we publish this?

File: modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc [hunk 6]
```diff
@@ -0,0 +1,310 @@
...
+[source, python]
+----
+path = 'teradatasql://'+ teradata_user +':' + teradata_password + '@'+host + '/?database=' +
+        teradata_database + '&LOGMECH=' + teradata_log_mech
+----
+== Run DBT
+Teradata Vantage has long been recognized as a database that provides superior ELT(Extract Load Transform) capabilities. DBT enhances this capabilities by making use of its efficient Analytic functions to transform data.
+So, in this case, we will use data tables, customers, accounts and transactions, that are made available on Vantage.
+
+=== Create the Dimension model
```
Comment (adamtwo) [inline modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc:238]: We do sentence capitalization for section titles

```suggestion
=== Create the dimensional model
```

File: modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc [hunk 7]
```diff
@@ -0,0 +1,310 @@
...
+ads_fv = FeatureView(name="ads_fv",entities=[customer],source=DBT_source, schema=[
+        Field(name="age", dtype=Float32),
+        Field(name="income", dtype=Float32),
+        Field(name="q1_trans_cnt", dtype=Int64),
+        Field(name="q2_trans_cnt", dtype=Int64),
+        Field(name="q3_trans_cnt", dtype=Int64),
+        Field(name="q4_trans_cnt", dtype=Int64),
+    ],)
+----
+=== Generate Training Data
```
Comment (adamtwo) [inline modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc:269]:
```suggestion
=== Generate training data
```

**Review by adamtwo — CHANGES_REQUESTED** [review]

**Comment by adamtwo** [comment]
Thanks for the PR, I left detailed comments at the file level. One general remark, the PR is adding the how-to to the ROOT module. In general, we try to keep the ROOT module thin. Instead, we try to have a layout where a module is owned by a single team. This will help us scale the review process in the future. We already have `modelops` and `other-integrations` modules. Please consider moving this content to these modules or creating a new module just for FEAST.

File: modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc (hunk 1 above)
Comment (chillanki) [reply modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc:4]: My full name is Ravi Chillanki 
Pen name is Ravi Chr :)

File: modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc (hunk 5 above)
Comment (chillanki) [reply modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc:46]: You are right Adam. Do you mean exactly : 'github.com' ? It is there in 'github.td.teradata.com' like i mentioned

**Pull request closed without merging**